string[] bot_id
nav_msgs/Odometry[] botpose
